Doc/whatsnew/3.11.rst

```diff
@@ -1884,6 +1884,17 @@ New Features
 Porting to Python 3.11
 ----------------------
 
+* Many macros have been converted to functions in the Python C API,
+  making them usable from other programming languages and avoiding
+  `macro pitfalls <https://gcc.gnu.org/onlinedocs/cpp/Macro-Pitfalls.html>`_
+  The change should be mostly transparent to users,
+  as the replacement functions will cast their arguments to the expected types
+  to avoid compiler warnings due to static type checks.
+  However, when the limited C API is set to >=3.11, these casts are not done,
```

Comment:
```suggestion
  However, when the limited C API is set to Python 3.11 or newer, these casts are not done,
```

Reply:
```suggestion
  However, when the limited C API is set to >=3.11,
  these casts are not done,
```

Added a line break to keep it under 80 columns